openstack/check.py

```diff
@@ -779,14 +796,11 @@ def check(self, instance):
             project = self.get_scoped_project(instance)
 
             # Restrict monitoring to non-excluded servers
-            excluded_server_ids = self.init_config.get("exclude_server_ids", [])
-            servers = list(
-                set(self.get_servers_managed_by_hypervisor()) - set(excluded_server_ids)
-            )
+            server_ids = self.get_servers_managed_by_hypervisor(instance)
```

Comment:
I'd be in favor of keeping it at the `init_config` level for two reasons:

1. That's how it originally worked in the code. Thus, keeping to do so means no backward incompatibility (in case a user found out about the inconsistency)
2. Network id exclusion is at the `init_config` level and it feels natural to have both at the same level.